main/meleeMenu.h

```diff
@@ -27,8 +28,10 @@ typedef struct
     font_t* font;
     meleeMenuCb cbFunc;
     uint8_t numRows;
+    uint8_t firstRowOnScreen;
     uint8_t selectedRow;
     uint8_t allowLEDControl;
+    uint8_t usePerRowXOffsets;
```

Comment:
if you're gonna use this like a `bool`, declare it as a `bool` and set it to `true` or `false`

Reply:
Also it's always set to `1`, so it doesn't seem particularly useful.

Reply:
`allowLEDControl` is used as a bool but defined as a uint8_t so I standardized on that.
I do actually set `usePerRowXOffsets` to 0 in my `nvs-manager` branch

Reply:
Because `allowLEDControl` is defined that way. Should both be defined as `bool` to match their usage?

Reply:
Assumed yes, implemented

Reply:
Yes, I was bad and no one reviews me :(